### PR TITLE
Add public console config to publish console URL

### PR DESCRIPTION
**Pull request by spadgett**
The global config resource is not readable by normal users. Add a public
config map that has the console URL so it can be discovered by all
authenticated users.

### CODE REVIEW

File: manifests/03-rbac-role-ns-openshift-config-managed.yaml [hunk 1]
```diff
@@ -12,3 +12,21 @@ rules:
...
   - list
   - watch
+  - create
+  - update
+  - delete
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-config-public
```
Comment (spadgett) [inline manifests/03-rbac-role-ns-openshift-config-managed.yaml]: @enj I followed the naming for our existing console operator roles here, but it looks like we should be using a `system:` prefix?

**Comment by spadgett** [comment]
Creates this config map, which is readable to any authenticated user. @deads2k @mfojtik fyi

```yaml
kind: ConfigMap
apiVersion: v1
metadata:
  name: console-config-public
  namespace: openshift-config-managed
  selfLink: /api/v1/namespaces/openshift-config-managed/configmaps/console-config-public
  uid: f4a85172-6c11-11e9-a140-060bde40ce6a
  resourceVersion: '1886802'
  creationTimestamp: '2019-05-01T13:06:38Z'
  labels:
    app: console
data:
  consoleURL: 'https://console-openshift-console.apps.spadgett.devcluster.openshift.com'
```

**Comment by spadgett** [comment]
Renamed the config map `console-public`

File: manifests/03-rbac-role-ns-openshift-config-managed.yaml [hunk 2]
```diff
@@ -12,3 +12,21 @@ rules:
   - get
   - list
   - watch
+  - create
```
Comment (enj) [inline manifests/03-rbac-role-ns-openshift-config-managed.yaml]: This role is too broad IMO.

**Comment by spadgett** [comment]
@enj updated

File: manifests/03-rbac-role-ns-openshift-config-managed.yaml (hunk 1 above)
Comment (enj) [reply manifests/03-rbac-role-ns-openshift-config-managed.yaml]: I think this is fine.  It does not escape the namespace and nothing else needs to bind to this role.

**Pull request merged**